src/sonic-host-services/scripts/determine-reboot-cause

```diff
@@ -98,18 +98,23 @@ def find_proc_cmdline_reboot_cause():
 
     return proc_cmdline_reboot_cause
 
+
 def get_reboot_cause_from_platform():
     # Until all platform vendors have provided sonic_platform packages,
     # if there is no sonic_platform package installed, we only provide
     # software-related reboot causes.
+    hardware_reboot_cause_major = str()
+    hardware_reboot_cause_minor = str()
     try:
         import sonic_platform
         platform  = sonic_platform.platform.Platform()
         chassis  = platform.get_chassis()
-        return chassis.get_reboot_cause()
+        hardware_reboot_cause_major, hardware_reboot_cause_minor = chassis.get_reboot_cause()
```

Comment:
not sure if this change is related to kvm or not, seems like a general change.

Reply:
That's correct. The error will be thrown for any platform which does have `sonic_platform` package installed. Modified the title accordingly.